packages/ui/src/machines/authenticator/index.ts

```diff
@@ -373,6 +374,7 @@ export function createAuthenticatorMachine() {
           event.data?.intent === 'confirmSignUp',
         shouldRedirectToResetPassword: (_, event) =>
           event.data?.intent === 'confirmPasswordReset',
+        shouldAutoSignIn: (_, event) => event.data?.intent === 'autoSignIn',
```

Comment:
I realize after few months of my refactor that `intent` is a bad name. Really what `intent` is the `redirectTarget`. `{ intent: 'autoSignIn' }` means that authenticator should go to `autoSignIn` state next.

Reply:
👍